FormFactory/AbstractMessageFormFactory.php

```diff
@@ -56,8 +56,6 @@ public function __construct(FormFactoryInterface $formFactory, AbstractType $for
      */
     protected function createModelInstance()
     {
-        $class = $this->messageClass;
-
-        return new $class();
+        return new $this->messageClass();
```

Comment:
I tested on PHP 5.3 and it works, I'm not sure it is completely safe everywhere but I'm quite confident.